docs/guide/hosting.md

```diff
@@ -0,0 +1,46 @@
+# Hosting Apostrophe in production
+
+## Server requirements
+
+We recommend hosting Apostrophe in self-hosted production environments with the following minimum specifications:
+
+| Software | Minimum Version |
+| ------------- | ------------- |
+| Ubuntu | 16+ |
+| [NGINX](https://www.nginx.com/) (or another proxy server) | Latest |
+| [Node.js](https://nodejs.org/en/) | 12.x+ |
+| [MongoDB](https://docs.mongodb.com/manual/administration/install-community/)  | 3.6+ |
+| [ImageMagick](https://imagemagick.org/script/download.php#macosx) (optional) | Any |
```

Comment:
A macosx link doesn't make sense here since this is production. Also it would be better to always guide people to pages that encourage installation via the package manager. ImageMagick looks like the main offender there because that would be downloading a binary RPM and manually installing it, which creates a maintenance burden. 

This should be sufficient on ubuntu, needs testing:

```
sudo apt update && sudo apt install imagemagick
```

The other links you gave should work out better for folks.

Reply:
That was from copying from the setting up page. I should remove it there as well.